contrib/checkbox-dss-validation/checkbox-provider-dss/units/test-plan.pxu

```diff
@@ -35,7 +35,11 @@ include:
     dss/create_tensorflow_cuda_notebook
     cuda/tensorflow_can_use_cuda
     dss/remove_tensorflow_cuda_notebook
+    dss/purge
+    dss_integration_tests/cpu
+    dss_integration_tests/nvidia_gpu
```

Comment:
In their respective job defintions, both `dss_integration_tests/cpu` and `dss_integration_tests/nvidia_gpu` depend on `dss/purge`, meaning they will be run after `dss/purge` has been run, but it doesn't mean they will both start from a "clean" state. As they are ordered here, the CPU test will run first, and whatever state DSS is in, it will be used right after for the Nvidia GPU test. Is that accounted for?

Reply:
yes ... the both integration tests reset the state back ... each of them first initializes DSS, runs some tests, then purges DSS.